src/Navigation/Link/styled.js

```diff
@@ -42,15 +42,14 @@ export const StyledNavLink = styled(NavLink).attrs(() => ({ activeClassName }))`
   padding: 14px 16px;
   border-radius: 24px;
   transition: 0.5s;
-  outline: none;
 
   &:hover {
     color: ${({ theme }) => theme.colors.black};
     background: ${({ theme }) => theme.colors.white};
   }
 
   &.${activeClassName} {
```

Comment:
Now it looks better, but after pressing button border jumps strangely. It's hard to describe, I attach a gif showing what I mean.
![outline](https://user-images.githubusercontent.com/83245340/130147421-7a3239a1-3c16-4304-a4b0-97f44bea32a8.gif)